eras/shelley/impl/src/Cardano/Ledger/Shelley/Rules/Reports.hs

```diff
@@ -50,9 +50,9 @@ showCred (KeyHashObj (KeyHash x)) = show x
 
 synopsisCert :: ShelleyEraTxCert era => TxCert era -> String
 synopsisCert x = case x of
-  ShelleyTxCertDeleg (RegKey cred) -> "RegKey " ++ take 10 (showCred cred)
-  ShelleyTxCertDeleg (DeRegKey cred) -> "DeRegKey " ++ take 10 (showCred cred)
-  ShelleyTxCertDeleg (Delegate _) -> "Delegation"
```

Comment:
Wonder why there is this difference in this case, of not showing the first 10, any idea?

Reply:
Probably oversight. This is only used in tests when assertions fail.